internal/app/handlers/shortener_handler.go

```diff
@@ -63,3 +66,59 @@ func (s *ShortenerHandler) ShortenerHandler(res http.ResponseWriter, req *http.R
 		return
 	}
 }
+
+type JSONRequest struct {
```

Comment:
Я бы предложил более информативное имя ведь типов запросов может быть много:
```suggestion
type ShortenerRequest struct {
```